docs/grid/cell-editing.md

```diff
@@ -64,14 +64,14 @@ Each column has a specific `dataType`, which can be set explicitly or inferred f
 
 In addition to `dataType` you can extend a selection of predefined validation rules:
 
-- `notEmpty`
-- `boolean`
-- `number`
-- `datetime`
-- `caseUnique`
-- `unique`
-
-To prevent users from entering `null` or empty string values in any column, add the `notEmpty` validation rule:
+- `notEmpty`- Only accepts non-empty values.
+- `boolean`- Only accepts `true`, `false`, `1` and `0` as valid values.
+- `number`- Only accepts numbers.
+- `datetime`- Only accepts valid timestamps.
+- `caseUnique`- Only accepts unique values within the column (case-insensitive).
```

Comment:
`caseUnique` name is very unintuitive to me, especially that `case` means case-insensitive. I know that long names should be avoided, but maybe we can accept something like `noCaseCheckUnique` or `ignoreCaseUnique`?

Reply:
Extend information about timestamps, as someone may use a UNIX timestamp.